qute/utilities/derive.py

```diff
@@ -10,18 +10,13 @@
 _NUMERIC_UI_MIN = _NUMERIC_UI_MAX * -1
 
 
-# ------------------------------------------------------------------------------
-def is_string(value):
```

Comment:
No need to do the python version test on demand, so simply resorted to basestring.
Would highly recommend adding `six` as a dependency instead though.